lib/reservation_generator_helper.rb

```diff
@@ -0,0 +1,96 @@
+module ReservationGeneratorHelper
+  require 'ffaker'
+  include ActiveSupport::Testing::TimeHelpers
+
+  def gen_res(random = false)
+    r = Reservation.new(status: 'reserved', reserver_id: User.all.sample.id,
+                        equipment_model: EquipmentModel.all.sample,
+                        notes: FFaker::HipsterIpsum.paragraph(2),
+                        start_date: Time.zone.today)
+    len = r.equipment_model.maximum_checkout_length
+    len -= random ? rand(len - 1) : 1
+    r.due_date = Time.zone.today + len.days
```

Comment:
I'd refer to `r.start_date` instead of explicitly calling `Time.zone.today` so that we only specify the start date onece.